app/src/main/java/com/cornellappdev/android/eateryblue/data/models/Eatery.kt

```diff
@@ -188,6 +189,103 @@ data class Eatery(
             }
         }
     }
+
+    fun formatOperatingHours(): List<Pair<String, List<String>>> {
```

Comment:
This is awesome; I really like the idea of returning some data structure like a `List<Pair<...>>`. I assume this is representing the label of each section paired to a list of its hours. Really great stuff!

...the only thing is that I had to infer that; I wouldn't have to infer it if you write spec! Please write a quick spec for all these functions/fields, even if they're private. The 2110 TA in me requires it.